plugins/powershell/help.md

```diff
@@ -188,6 +188,7 @@ Invoke-Expression ((New-Object System.Net.Webclient).DownloadString('https://raw
 
 # Version History
 
+* 3.0.3 - Fix decoding error in `Powershell String` action and `Execute Script` action | Update SDK | Update `libffi6` package to `libffi7`
```

Comment:
Are there any changes to the `action.py` files? Just asking as I don't see any in the diff 😄

Reply:
Yes, thanks for noticing - I failed to commit. Please hold off the review for now if possible though, there is one query being resolved beforehand. I will update the PR status.